bumble/hci.py

```diff
@@ -701,29 +703,22 @@ def phy_list_to_bits(phys: Optional[Iterable[int]]) -> int:
 
 
 # ACL
```

Comment:
This comment line should be moved now to line 712

Reply:
Done.